okhttp/src/main/kotlin/okhttp3/internal/connection/RealCall.kt

```diff
@@ -87,10 +87,10 @@ class RealCall(
 
   var connection: RealConnection? = null
     private set
-  private var exchangeRequestDone = false
-  private var exchangeResponseDone = false
+  private var requestBodyOpen = false
+  private var responseBodyOpen = false
```

Comment:
these booleans are inverted. I didn’t invert the initial values because it doesn’t matter until we initialize the first exchange.